recipes/rapidpe-rift-pipe/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "rapidpe-rift-pipe" %}
+{% set version = "0.0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rapidpe_rift_pipe-{{ version }}.tar.gz
+  sha256: 16e31facbd2caf78563e74177f045a2cfef281a273156f109aef6e4ab0c05189
+
+build:
+  entry_points:
+    - rapidpe-rift-pipe = rapidpe_rift_pipe.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```

Since this looks to be pure Python, better to make this recipe noarch (and we can ignore the failure on Windows due to missing dependencies:
```suggestion
  noarch: python
```